crates/uv-resolver/src/error.rs

```diff
@@ -248,6 +253,51 @@ impl std::fmt::Display for NoSolutionError {
     }
 }
 
+#[allow(clippy::print_stdout)]
+fn display_tree(error: &DerivationTree<PubGrubPackage, Range<Version>, UnavailableReason>) {
+    let mut lines = Vec::new();
+    display_tree_inner(error, &mut lines, 0);
+    lines.reverse();
+    println!("UV_INTERNAL__SHOW_DERIVATION_TREE\n{}", lines.join("\n"));
```

Comment:
Maybe to stderr?

Reply:
It's nice that it's separate from the rest of the command output right now, but yeah that's a logical place for it. Hm.